06.09.2018 15:14:27 ERROR JdbcWithLog4j:33 - there is an ERROR with connection
06.09.2018 15:14:27 ERROR JdbcWithLog4j:34 - IO Error: The Network Adapter could not establish the connection
